tools/FakeItEasy.Build/Program.cs

```diff
@@ -83,14 +86,14 @@
         // expect the line to look something like
         // requires-python = "~=x.yz"
         // . We want the x.yz part.
-        const string versionSourceFile = "pyproject.toml";
+        const string versionSourceFile = "docs/pyproject.toml";
         const string versionPattern = @"requires-python = ""~=(?<version>[^""]+)""";
         var pythonVersion = File.ReadLines(versionSourceFile)
             .Select(line => Regex.Match(line, versionPattern))
             .Where(m => m.Success)
             .Select(m => m.Groups["version"].Value)
             .FirstOrDefault() ?? throw new InvalidOperationException($"Could not find required Python version in {versionSourceFile}");
-        File.WriteAllText(".python-version", pythonVersion);
+        File.WriteAllText("docs/.python-version", pythonVersion);

```

Comment:
we're lucky the `.gitignore` file didn't root the path to this file, something I noticed when I added it, and didn't care for, but decided to just go with the flow.